replica/replica.go

```diff
@@ -975,7 +979,11 @@ func (r *Replica) createDisk(name string, userCreated bool, created string) erro
 	if newSnapName != "" {
 		r.addChildDisk(newSnapName, newHeadDisk.Name)
 		r.diskData[newSnapName] = r.diskData[oldHead]
-		r.diskData[newSnapName].Name = newSnapName
+		if userCreated {
+			r.diskData[newSnapName].Name = fmt.Sprintf(diskName, name)
```

Comment:
It's not necessary that user provide snapshot name while creation, could you verify `./jivactl snapshot create` commands ?